lib/pymedphys/docs/release-notes.md

```diff
@@ -8,7 +8,7 @@
 
 # Release Notes
 
-All notable changes to this project will be documented in this file.
+All notable changes to are documented here.
```

Comment:
We changed the reference file to instead be the README.rst file at the root of PyMedPhys. I find it by default makes much more sense for new contributors. I suspect you'll need to make sure you're editing the reference file (at the root directory)

Reply:
Ah yes, I remembered that for a different change (fixing a broken link) but forgot it here. Cheers.

Reply:
wait, I _did_ make this change in the CHANGELOG? Or did you?

Reply:
Yeah, I got confused I think. All good :slightly_smiling_face: